docs/components/modeler/forms/form-element-library/forms-element-library-datetime.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: forms-element-library-datetime
 title: Datetime
-description: A form element to read and edit date and time data
+description: Learn about the datetime form element to read and edit date and time data.
```

Comment:
It's merged so it's perhaps a little late but this does make the descriptions inconsistent across the individual components, some starting with 'Learn about {X}' and others with 'A'. 

The change makes sense but I feel like perhaps it should be applied across the board?

Reply:
I don't disagree here. Ideally, descriptions should include action items/verbs for SEO purposes, hence the "learn about...".

Monthly I've been updating descriptions flagged as too short or long, and I do have a long-term goal of cleaning up all of the descriptions across documentation entirely. I'll mark this as an issue on my end for the backlog 👍